peachjam/templates/peachjam/_court_list.html

```diff
@@ -12,6 +12,10 @@ <h4 class="mb-2">{{ court_class }}</h4>
               </li>
             {% endfor %}
           </ul>
+          {% if court_class.show_listing_page %}
+            <a class="btn btn-sm btn-primary"
```

Comment:
I recommend we make this a simple link below the h4 heading, with `class="mb-3"` and text "All judgments"

Reply:
good idea. this is how it looks now
![image](https://github.com/laws-africa/peachjam/assets/25079238/baeffbe2-f603-46b5-9b5e-20f8435d9695)